.circleci/config.yml

```diff
@@ -122,9 +122,9 @@ jobs:
                 exit 0
             fi
 
-            # Run flake8 on all excluding Integraions and Scripts (they will be handled in linting)
+            # Run flake8 on all excluding Packs (Integraions and Scripts) - they will be handled in linting
             ./Tests/scripts/pyflake.sh *.py
-            find . -maxdepth 1 -type d -not \( -path . -o -path ./Integrations -o -path ./Scripts -o -path ./Beta_Integrations -o -path ./venv \) | xargs ./Tests/scripts/pyflake.sh
+            find . -maxdepth 1 -type d -not \( -path . -o -path ./Packs -o -path ./venv \) | xargs ./Tests/scripts/pyflake.sh
```

Comment:
we should still run on `Beta_Integrations`

Reply:
this line specified on what directories NOT to run. As we don't have this dir any more it is not needed.